Makefile

```diff
@@ -168,9 +189,11 @@ pip-pytest: pip
 prospector: pip-prospector
 	prospector $(HUB_SRC_DIR)
 
-# testing GitHub connection from terminal to GitHub 
+#Rebuilds the Makefile if any of the configuration files have changes or if
```

Comment:
Do we have a Makefile.in? Do we have a config.status file? What is this target doing?

Reply:
I took it as it would store the config.status into hub directory but i look at it now and youre right we dont have a makefile.in